### PR TITLE
only one commit, because it's a timed test.  Everything looks done

### PR DESCRIPTION
**Pull request by chuckie512**

### CODE REVIEW

**Comment by chuckie512** [comment]
I think i might have messed something up with the PRs.  look at my repo if you'd like

File: template/quiz2.html [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
```
Comment (matty-digital) [inline template/quiz2.html]: not necessary with the html5 spec

File: template/quiz2.js [hunk 2]
```diff
@@ -1,39 +1,87 @@
...
+
+
+	//on mouse over
+	$mouseover.mouseover( function() {
 
-	$mouseover.on('mouseover', function() {
 		$this = $(this);
 		$(this).html('Scrooge McDuck!');
-		$(this).height($(this).height() + 50);
+		$(this).height($(this).height() + 50); //why would you make it bigger every time!?!??!?!?!?!?!?
```
Comment (matty-digital) [inline template/quiz2.js]: I'm going to go with...becuase

File: template/quiz2.js [hunk 3]
```diff
@@ -1,39 +1,87 @@
...
 		}
 	});
 
-	$(document).on(ready, function() {
-		setTimeout({
-			$timeout.fadeIn('slow');
-		}, 1000);
-	});
+    $button.click(function(){
+        if($(this).html()=="Get title") {
```
Comment (matty-digital) [inline template/quiz2.js]: `===` for this check

**Pull request closed without merging**